internal/model/error.go

```diff
@@ -42,6 +42,12 @@ type Error struct {
 	// This may be set when a stack trace cannot be parsed.
 	StackTrace string
 
+	// LogMessage holds ECS compatible error message for log event type.
+	LogMessage string
```

Comment:
```suggestion
	// Message holds an error message.
	//
	// Message is the ECS field equivalent of the APM field `error.log.message`.
	Message string
```

Probably as a followup: we should think about setting `error.message` wherever we currently set `error.log.message`, and add an alias from `error.log.message` to `error.message`. See also https://github.com/elastic/apm-server/issues/3878